src/patterns/check-answers/index.md.njk

```diff
@@ -28,6 +28,14 @@ Check answers pages help to:
 - increase users’ confidence as they can clearly see that they have completed all the sections and that their data has been captured
 - reduce error rates as users are given a second chance to notice and correct errors before submitting data
 
+### Choose the right layout for the page
+
+Many services ask users for short answers, which are suitable for a layout that takes up two-thirds of the screen on desktop devices. For example, the 'Check your answers before submitting your application' page included in this guidance.
+
+Using a two-thirds layout stops lines of text getting so long that the page becomes difficult to read. [To learn more about line length, see ['Layout'](/styles/layout/). It also means the action links are closer to the other content on the page. This means users with screen magnifiers are less likely to miss the links.
```

Comment:
Can we merge these two sentences?
```suggestion
Using a two-thirds layout stops lines of text getting so long that the page becomes difficult to read. [To learn more about line length, see ['Layout'](/styles/layout/). It also means the action links are closer to the other content on the page and users with screen magnifiers are less likely to miss them.
```

Reply:
@calvin-lau-sig7 I don't think it would do any harm, but 26 words is [longer than we recommend](https://insidegovuk.blog.gov.uk/2014/08/04/sentence-length-why-25-words-is-our-limit/)...

Reply:
Ahh I just didn't like the feel of that repetition. How about we just split it (edited above)?